packages/random-id/src/get-random-id.injectable.ts

```diff
@@ -5,10 +5,8 @@
 import { getInjectable } from "@ogre-tools/injectable";
 import { v4 as getRandomId } from "uuid";
 
-const getRandomIdInjectable = getInjectable({
+export const getRandomIdInjectable = getInjectable({
```

Comment:
@jansav I could use some help with adding getInjectionToken here

Reply:
Sure. 

```typescript
import { getInjectable, getInjectionToken } from "@ogre-tools/injectable";
import { v4 as getRandomId } from 'uuid';

export type GetRandomId = () => string;

export const getRandomIdInjectionToken = getInjectionToken<GetRandomId>({
  id: "get-random-id-injection-token",
});

export const getRandomIdInjectable = getInjectable({
  id: "get-random-id",
  instantiate: (): GetRandomId => getRandomId,
  injectionToken: getRandomIdInjectionToken,
  causesSideEffects: true
});
```

Reply:
typing is as ```() => string``` does not work.  

neither does 

```
<string, void>
<string, unknown>
<unknown>
<string, unknown, any>
```

and even when I pull in the defs from uuid
```
export type OutputBuffer = ArrayLike<number>;
export type v4String = (options?: V4Options) => string;
export type v4Buffer = <T extends OutputBuffer>(options: V4Options | null | undefined, buffer: T, offset?: number) => T;
export type v4 = v4Buffer & v4String;

... = getInjectionToken<v4> ...
```